docs/Date/index.md

```diff
@@ -0,0 +1,36 @@
+# Date
+A simple utility class for calculating time differences and rudimentry outputting.
+
+## Comparision
+
+Two date objects can be compared.
+
+### Example
+
+```csharp
+CF_Date dateA = CF_Date.CreateDateTime(2021, 12, 2, 10, 24, 3);
+CF_Date dateB = CF_Date.CreateDateTime(2021, 12, 4, 10, 24, 3);
+Print(dateB.Compare(dateA).GetDay());
+// "2"
```

Comment:
I am not sure what the intent here is? Comparing dates should usually return some kind of TimeSpan, as in they differ by 2 Days or something. Returning a new date and call GetDay there feels super strange. It's comparing apples to oranges.

Maybe have a look at how C# handles DateTime and TimeSpan. But it's a supermassive feature. I am not sure if you wanted to touch that here.

Reply:
I will remove Compare from this PR and add it back in a later PR, it is a new feature but not required for any other PRs.